upgrade/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "upgrade-xdai",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
```

Comment:
the license of the entire repo is GPL-3. Do you think the license of the scripts should differ?

Reply:
Sorry, this was autogenerated. Fixed https://github.com/poanetwork/tokenbridge-contracts/pull/337/commits/30b4fa909200ab60185ca11f591fa3324d3c5a6c